libpod/pod.go

```diff
@@ -696,3 +710,35 @@ func (p *Pod) Status() (map[string]ContainerStatus, error) {
 // TODO add pod batching
 // Lock pod to avoid lock contention
 // Store and lock all containers (no RemoveContainer in batch guarantees cache will not become stale)
+
+// Inspect returns a PodInspect struct to describe the pod
+func (p *Pod) Inspect() (PodInspect, error) {
+	var (
+		podContainers []PodContainerInfo
+	)
+
+	containers, err := p.AllContainers()
+	if err != nil {
+		return PodInspect{}, err
+	}
+	for _, c := range containers {
+		containerStatus := "uknown"
```

Comment:
s/uknown/unknown ?

Reply:
this is the only blocking one, I am new to this github reviewing business.

Reply:
fixed